change/@fluentui-react-charting-3fdcb2d7-fde5-4ec3-bf0e-1e9d32cd1194.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Changes for lazy load for hover card and legends",
+  "packageName": "@fluentui/react-charting",
+  "email": "ankityadav@microsoft.com",
```

Comment:
Has the charting bundle changed in structure or size after lazy loading

Reply:
Master Bundle:
{"sizes":{"fluentui-react-charting-AreaChart":240372,"fluentui-react-charting-CommonComponents":486,"fluentui-react-charting-DonutChart":179983,"fluentui-react-charting-GaugeChart":185718,"fluentui-react-charting-GroupedVerticalBarChart":224582,"fluentui-react-charting-HeatMapChart":223706,"fluentui-react-charting-HorizontalBarChart":141558,"fluentui-react-charting-HorizontalBarChartWithAxis":228708,"fluentui-react-charting-Legends":153396,"fluentui-react-charting-LineChart":264222,"fluentui-react-charting-PieChart":68265,"fluentui-react-charting-SankeyChart":147061,"fluentui-react-charting-Sparkline":84383,"fluentui-react-charting-StackedBarChart":173708,"fluentui-react-charting-TreeChart":81108,"fluentui-react-charting-VerticalBarChart":233728,"fluentui-react-charting-VerticalStackedBarChart":231954}}

LazyLoaded-Bundle:
{"sizes":{"5":31179,"15":2723,"fluentui-react-charting-AreaChart":246003,"fluentui-react-charting-CommonComponents":486,"fluentui-react-charting-DonutChart":179983,"fluentui-react-charting-GaugeChart":185718,"fluentui-react-charting-GroupedVerticalBarChart":230924,"fluentui-react-charting-HeatMapChart":230121,"fluentui-react-charting-HorizontalBarChart":141558,"fluentui-react-charting-HorizontalBarChartWithAxis":235519,"fluentui-react-charting-Legends":153396,"fluentui-react-charting-LineChart":244368,"fluentui-react-charting-PieChart":68265,"fluentui-react-charting-SankeyChart":147061,"fluentui-react-charting-Sparkline":84383,"fluentui-react-charting-StackedBarChart":173708,"fluentui-react-charting-TreeChart":81108,"fluentui-react-charting-VerticalBarChart":240745,"fluentui-react-charting-VerticalStackedBarChart":238931}}

I will try to gather more information on this. Created the bug for that https://uifabric.visualstudio.com/iss/_workitems/edit/8950/